modules/test/playwright/pages/commerce/commerceCatalogSystemSettingsPage.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {SystemSettingsPage} from '../configuration-admin-web/SystemSettingsPage';
+
+export class CommerceCatalogSystemSettingsPage {
+	readonly page: Page;
+	readonly enabledButton: Locator;
+	readonly systemSettingsPage: SystemSettingsPage;
+	readonly updateButton: Locator;
+
+	constructor(page: Page) {
+		this.enabledButton = page.getByLabel('enabled');
+		this.systemSettingsPage = new SystemSettingsPage(page);
+		this.updateButton = page.getByRole('button', {name: 'Update'});
+	}
+
+	async configureProductVersioning() {
```

Comment:
I would call this toggleProductVersioning rather then configureProductVersioning